<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="typeBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>int</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>double</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>string</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IPv4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IPv6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MAC</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="listBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Linked list</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Array list</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vector list</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sortBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Quick sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Insertion sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Merge sort</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="insertButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="randButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rand</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
